scripts/dev_setup.sh

```diff
@@ -13,6 +13,7 @@
 
 # fast fail.
 set -eo pipefail
+set -x
```

Comment:
fwiw I usually put the line below  in all .sh right after the `set -eo pipefail`, this way I can always turn -x on without script changes, with `TRACE=1 script` or just env (e.g. in CI). Just a convenience.

```sh
  [[ -v TRACE && "$TRACE" ]] && set -x	# to turn trace on,  pass TRACE=1 from upstairs
```